src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,92 @@
+.product-card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
```

Comment:
Better to set this one time for all elements under * selector in normalize file

Reply:
Check this comment, looks like you didn't fix it